README.md

```diff
@@ -27,7 +27,7 @@ pip install mirdata
 import mirdata
 import random
 
-orchset = mirdata.Dataset('orchset')
+orchset = mirdata.load('orchset')
```

Comment:
thinking about this. I totally agree with `load` being perceived maybe as loading to memory. trying to come up with a solution. `get` is a bit vague, and sounds like download a bit. `dataset` is confusing, because we already have `Dataset` the object and `datasets` the module. I'm worried about adding a third `dataset` the function.

I think i'll go with `initialize`? Unless anyone has another idea!

Reply:
Yeah, I think `initialize` is a good idea! I'll help to avoid confusion with the word "dataset" around. Unless anyone proposes a different name, I'll arrange it on my docs PR to save time!